server/api/recommendations.ts

```diff
@@ -0,0 +1,17 @@
+export default defineEventHandler(async (event) => {
+  try {
+    const { locale } = getQuery(event)
+
+    const recommendations = await fetch(
+      `https://firefox-api-proxy.readitlater.com/desktop/v1/recommendations?consumer_key=moso-web-dev&locale=${locale}`,
```

Comment:
Just curious - what does this `consumer_key` do?

Reply:
It's a unique identifier/hash that platforms would include when making requests, and we'd track it on the backend to know which platform was making the request. 

3rd party developers could apply to get their own, though they would have some limits (number of requests per day, can't hit certain endpoints, etc). 

This key was recently removed as a requirement for this endpoint, hence why it's now text based.

More info on the consumer key and our 3rd party developer program is here:
https://getpocket.com/developer/docs/authentication

Reply:
> This key was recently removed as a requirement for this endpoint, hence why it's now text based.

Aha! I was wondering about this. Thanks for the extra context!